packages/app/playwright/e2e/CreateWallet.test.ts

```diff
@@ -35,20 +35,20 @@ test.describe('CreateWallet', () => {
     await getButtonByText(page, /I accept/i).click();
 
     /** Copy Mnemonic */
-    await hasText(page, /Backup your Recovery Phrase/i);
+    await hasText(page, /Backup your Seed Phrase/i);
     await getButtonByText(page, /Copy/i).click();
     const savedCheckbox = await getByAriaLabel(page, 'Confirm Saved');
     await savedCheckbox.click();
     await getButtonByText(page, /Next/i).click();
 
     /** Confirm Mnemonic */
-    await hasText(page, /Enter your Recovery Phrase/i);
+    await hasText(page, /Enter seed phrase/i);
```

Comment:
I think we should be more consistent here, and either choose to use "your" with everything, like "Backup your Seed Phrase" and "Enter your seed phrase", or without "your" like "Backup Seed Phrase" and "Enter seed phrase".

Reply:
Can't always fit "your" in the phrases like `I have a 12 words Seed Phrase`, so will be dropping "your" entirely and stick with "Seed Phrase" instead.

Reply:
For me, reading the texts with "your" removed makes the texts sound very incorrect, and weird.